tests/e2e/chaos/chainlink-relay-terra.yaml

```
@@ -0,0 +1,33 @@
+namespace_prefix: chainlink
+charts:
+  localterra:
+    index: 1
+  mockserver-config:
+    index: 2
+  mockserver:
+    index: 3
+  chainlink:
+    index: 4
+    values:
+      replicas: 5
+      chainlink:
+        image:
+          image: "public.ecr.aws/z0b1w9r9/chainlink"
+          version: "candidate-develop-terr-qa-test-b64-decode-signers-3.233d4d167de902af193b8c83cc79e707d7e74541"
```

`release/develop-terr-qa-test-feb-02`
https://github.com/smartcontractkit/chainlink/compare/release/develop-terr-qa-test-feb-02
```suggestion
          version: "candidate-develop-terr-qa-feb-02"
```